### PR TITLE
Base ReportedError on OscBaseError instead of RuntimeError

### PR DESCRIPTION
**Pull request by mimi1vx**
This will properly print exception message when exception happens

### CODE REVIEW

File: oscqam/errors.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-class ReportedError(RuntimeError):
+from osc.oscerr import OscBaseError
+
+
+class ReportedError(OscBaseError):
```
Comment (kalikiana) [inline oscqam/errors.py:4]: Do you have an example where this makes a difference?

Comment (mimi1vx) [reply oscqam/errors.py:4]: any command which throws an exception :D

Comment (mimi1vx) [reply oscqam/errors.py:4]:
~~~qam assign SUSE:Maintenance:28561:294030
Traceback (most recent call last):
  File "/usr/bin/osc", line 33, in <module>
    sys.exit(load_entry_point('osc==1.1.1', 'console_scripts', 'osc')())
  File "/usr/lib/python3.10/site-packages/osc/babysitter.py", line 211, in main
    sys.exit(run(commandline.OscMainCommand()))
  File "/usr/lib/python3.10/site-packages/osc/babysitter.py", line 60, in run
    prg.main(argv)
  File "/usr/lib/python3.10/site-packages/osc/commandline.py", line 527, in main
    cmd.run(args)
  File "/usr/lib/python3.10/site-packages/osc/commandline.py", line 236, in run
    cmd.run(args)
  File "/usr/lib/osc-plugins/qam_assign.py", line 49, in run
    action()
  File "/usr/lib/python3.10/site-packages/oscqam/actions/oscaction.py", line 33, in __call__
    return self.action(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/oscqam/actions/assignaction.py", line 94, in action
    group = self.reviewable_group()
  File "/usr/lib/python3.10/site-packages/oscqam/actions/assignaction.py", line 103, in reviewable_group
    groups = self.user.reviewable_groups(self.request)
  File "/usr/lib/python3.10/site-packages/oscqam/models/user.py", line 48, in reviewable_groups
    raise NoQamReviewsError(reviews)
oscqam.errors.NoQamReviewsError: No 'qam'-groups need review.

Comment (mimi1vx) [reply oscqam/errors.py:4]: @kalikiana -- `osc.babysitter` has a huge list of caught exceptions. In past, with redefined logger and stderr/stdout it worked without problems. But with new class cmd plugins/commands, we need inherit one of Exceptions defined in osc itself

**Review by perlpunk — APPROVED** [review]

**Pull request merged**